src/RepoBackend.ts

```diff
@@ -345,6 +345,17 @@ export class RepoBackend {
         });
         break
       }
+      case "DocumentMessage": {
+        const { contents, id } = (msg as DocumentMsg);
+        this.meta.docsWith(id).forEach( (doc: string) => {
```

Comment:
Not sure if this is the right thing to do. Might need some further thought.

Reply:
This seems incorrect - supposedly we're trying to send a message to a particular document, not the set of documents this actor is participating in, each of which has its own document id. We should either only send messages to the document specified by `id` here, or change the message semantics to reflect the intent to send the message to all documents this actor is participating in. 

More of "actor broadcast"? Don't have a use case in mind for that atm.

Reply:
I have done this.